app/forms/questionnaires/qualified_teacher_check.rb

```diff
@@ -177,7 +177,7 @@ def ni_number_required?
     attr_reader :verified_trn
 
     def trn_to_store
-      if trn_verified?
+      if trn_verified? && verified_trn.present?
```

Comment:
I would say that if we have `trn_verified?` set to true, it would implied that we have `trn` present. Maybe it would be worth to rework this bit?